spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -9,18 +9,74 @@ It covers topics such as creating and running task applications.
 If you are just starting out with Spring Cloud Data Flow, you should probably read the "`<<getting-started.adoc#getting-started, Getting Started>>`" guide before diving into this section.
 --
 
-
-
 [[spring-cloud-dataflow-task-intro]]
 == Introduction
 
 A task executes a process on demand.
 In the case of Spring Cloud Task, a task is a http://projects.spring.io/spring-boot/[Spring Boot] application that is annotated with `@EnableTask`.
-A user launches a task that performs a certain process, and, once complete, the task ends.
+A user launches a task that performs a certain process, and, once complete, the task ends.  Unlike a stream where a stream definition can have at most one deployment a single task definition can be launched multiple times simultaneously.
 An example of a task would be a Spring Boot application that exports data from a JDBC repository to an HDFS instance.
 Tasks record the start time and the end time as well as the boot exit code in a relational database.
 The task implementation is based on the http://cloud.spring.io/spring-cloud-task/[Spring Cloud Task] project.
 
+The general lifecycle of a Task is:
+
+. Register Applications.
+. Create Task Definition.
+. Launch Task.
+
+You can use the Local Data Flow Server to launch tasks to your local machine, the Data Flow Server for Cloud Foundry to launch tasks to a single org and space on Cloud Foundry.
+Similarly, you can use Data Flow Server for Kuberenetes to launch a task to a single namespace on a Kubernetes cluster.
+See http://cloud.spring.io/spring-cloud-dataflow/#platform-implementations[the Spring Cloud Data Flow project page] for a list of Data Flow server implementations.
+
+NOTE: Launching Spring Cloud Task applications can not be delegated to Skipper at this time.  You must use the Data Flow Server option to launch tasks.
+
+[[spring-cloud-dataflow-stream-intro-dsl]]
+=== Launching a task
+
+A task is defined by specifying the Task application you wish to execute along with any properties that you want to remain static for the definition.
+Each Spring Cloud Task application is registered under a simple name.
+The registration process specifies where the application can be obtained (for example, in a Maven Repository or a Docker registry).  You can find out more information on how to register Spring Cloud Task applications in this <<spring-cloud-dataflow-register-task-apps,section>>.
```

Comment:
Referring to the 'how to register apps' sections is too confusing.  In this getting started section, you should show how to register the timestamp task, just as we show how to register `http` source and `log` sink for streams.

Reply:
Actually, this whole section for getting started should be moved up into the `getting-started.adoc` section.  

Look at https://docs.spring.io/spring-cloud-dataflow-server-cloudfoundry/docs/1.3.0.RELEASE/reference/htmlsingle/#getting-started-deploying-tasks

and 

https://docs.spring.io/spring-cloud-dataflow-server-kubernetes/docs/1.3.1.RELEASE/reference/htmlsingle/#_deploying_tasks

for reference